### PR TITLE
PFW-1434: MK3 Temp model calibration during wizard

### PR DESCRIPTION
**Pull request by 3d-gussner**
Add Temp model calibration to Wizard

- Unhack Selftest hack during Wizard
- Update `pot` and `po` files
  - Translated messages 
    - [x] CS with Deepl
    - [x] DE
    - [x] ES with Deepl
    - [x] FR with Deepl
    - [x] IT with Deepl
    - [x] PL with Deepl
    - Community translators please translate these two messages asap.
    - [X] NL @vintagepc please review
    - [x] RO @Hauzman 
    - [x] HU @AttilaSVK 
    - [ ] HR  @Prime1910 
    - [x] SK @ingbrzy 
    - [ ] SV @Painkiller56 
    - [x] NO @OS-kar

Tests
- [X] Prusa MK3S without MMU2 Factory reset All Data
  - [X] Wizard
  - [X] Selftest
  - [X] Reset printer during Wizard Temp model calibration
    - [X] Recover with Wizard at Temp model calibration
- [X] Prusa MK3S without MMU2 Factory reset Prep Shipping
- [X] Prusa MK3S with MMU2 Factory reset All Data
  - [X] Wizard
  - [X] Selftest
  - [X] Reset printer during Wizard Temp model calibration
    - [X] Recover with Wizard at Temp model calibration

- [X] Prusa MK2.5S without MMU2 Factory reset All Data

### CODE REVIEW

**Review by vintagepc — APPROVED** [review]

**Comment by ingbrzy** [comment]
I have made PR to this https://github.com/3d-gussner/Prusa-Firmware/pull/85

File: Firmware/Configuration.h [hunk 1]
```diff
@@ -550,7 +550,12 @@ enum CalibrationStatus
 	// For the wizard: factory assembled, needs to run Z calibration.
 	CALIBRATION_STATUS_Z_CALIBRATION = 240,
 
-	// The XYZ calibration has been performed, now it remains to run the V2Calibration.gcode.
+#ifdef TEMP_MODEL
+	// The XYZ calibration has been performed, needs to run Temp model calibration.
+	CALIBRATION_STATUS_TEMP_MODEL_CALIBRATION = 235,
+#endif //TEMP_MODEL
+
+// The XYZ calibration AND OR Temp model calibartion has been performed, now it remains to run the V2Calibration.gcode.
```
Comment (DRracer) [inline Firmware/Configuration.h]: /nitpick/: typo: calibartion -> calibration

File: Firmware/ultralcd.cpp [hunk 2]
```diff
@@ -4038,6 +4049,13 @@ void lcd_wizard(WizState state)
 			}
 			else end = true;
 			break;
+#ifdef TEMP_MODEL
+		case S::TempModel:
+			lcd_show_fullscreen_message_and_wait_P(_i("Temp model cal. takes apporx. 12 mins."));////MSG_TM_CAL c=20 r=4
```
Comment (DRracer) [inline Firmware/ultralcd.cpp]: typo: apporx -> approx
that affects the POT/PO files as well

**Comment by 3d-gussner** [comment]
@Prime1910 @Hauzman @Painkiller56 @AttilaSVK Can you please provide the translation for
- EN: `Temp model not calibrated yet.`
- EN: `Temp model cal. takes approx. 12 mins.`

**Comment by Hauzman** [comment]
@3d-gussner Hello i will try to make it invthe weekend if it's not too late

**Comment by 3d-gussner** [comment]
> @3d-gussner Hello i will try to make it in the weekend if it's not too late

Sooner is always better but it is okay if we get this bit later.

**Review by DRracer — APPROVED** [review]

**Comment by Hauzman** [comment]
@3d-gussner 
> * EN: `Temp model not calibrated yet.`
  Ro: Modelul de temp. nu este inca calibrat.
> * EN: `Temp model cal. takes approx. 12 min`
  Ro: Cal. Model de temp dureaza aprox. 12 min

**Comment by 3d-gussner** [comment]
- [x] @AttilaSVK please review https://github.com/prusa3d/Prusa-Firmware/pull/3769/files#diff-3a7d51d07fb16c60d273b7986aa88a7892939b19a9bad4f3501a153ffb0604e9
- [ ] @Prime1910 please review https://github.com/prusa3d/Prusa-Firmware/pull/3769/files#diff-487d5469566575627c7dc47abf437c52e890de941c1fd8d4db5a2a4d6ef08d64
- [ ] @Painkiller56 please review https://github.com/prusa3d/Prusa-Firmware/pull/3769/files#diff-cf46655e0192f28112da2985d28dc728949010730a0ac629dbb7049f9f53a357

**Review by AttilaSVK — APPROVED** [review]
Hungarian translation is fine

**Comment by 3d-gussner** [comment]
> Hungarian translation is fine

Thanks

**Pull request merged**